Information/Design.md

```diff
@@ -89,3 +89,320 @@
+out nothing to the user via Discord. 
+
+When this command is entered, it will first prompt the user for the user's account name. This 
+way each user has its own saved list of monsters killed. If the user does not have a valid 
+account name or id then the monster is not saved in any list, and an appropriate message is 
+displayed to the user via discord. The user will have to authenticate themselves via their 
+password as well. This will be the same password as in the Discord server. This way random 
+people belonging to the server this Monster Hunter World bot is associated with. Cannot 
+edit the list of monsters they have killed without the knowledge of the users pass code.
+
+There are 48 total monsters in the Monster Hunter World video game. Thus the maximum size 
+of any monster list will be 48. The smallest size of course would be if there were no 
+monsters killed, thus size = 0. Being how the upper bound of a monster list is 48. 
+printing out to discord should not cause any significant undefined bandwidth issues. 
+
+Potentially I could have returned a dictionary here instead of a list, however,
+you would have to utilize a value, or key field per cell in a dictionary in the python 
+programming language. Since it is not needed and would only cause of more unneeded data 
+to the user, and in code. It simply is complexity that is not needed. Thus a list or array 
+in the python language is best for this design.  
+
+
+examples of _my victims_ command usage:
+
+`-my victims Kulu-Ya-Ku`  -- example with _name_ parameter 
+
+
+example of the _user entry_ aspect of the _my victims_ command:
+
+`Enter user account name: Josh`
+
+`Enter the password for Josh: ********`
+
+------------------------------------------------------------------------------------------
+
+
+######REQ 170 Design:
+
+
+Command name: _-gone to soon_ 
+
+Parameters: _name_ 
+
+**Design of _user_ command**:
+
+
+This command is the exact same design as the **REQ 160**. In every way the only difference
+is the command name is different. The design and implementation will be identical.
+Therefore, refer to **REQ 160** to find out the details. 
+
+
+
+examples of _gone to soon_ command usage:
+
+`-gone to soon Kulu-Ya-Ku`  -- example with _name_ parameter 
+
+
+example of the _user entry_ aspect of the _gone to soon_ command:
+
+`Enter user account name: Josh`
+
+`Enter the password for Josh: ********`
+
+
+
+######REQ 180 Design:
+
+
+Command name: _-help_ 
+
+Parameters: None
+
+**Design of _help_ command**:
+
+The _help_ command will list all available commands supported by the bot. Moreover will 
+print a brief description of each command. It will do this by printing a dictionary to 
+Discord Server. The key in the dictionary is the command name, the value shall be a brief 
+description or purpose of the command.  
+
+In total, the number of commands are 18 for this discord bot. Thus the size of this data 
+structure is 18 cells in the command dictionary. 
```

Comment:
The number of commands may change, don't specificy size. Also python doesn't require size defined, so why would we care?

Reply:
Ok, I have changed this aspect of the design already. I molded your design advice, with mine. I hope it is ok.